### ISSUE TITLE
Title Background not applied

### ISSUE THREAD

**Issue by peter-lyons-kehl** [oreporan/color-tabs-vscode#74]
Thank you for ColorTabs. Confirming that coloring of the bottom of the tabs works (other than #72).

However, the tab background itself doesn't seem to change. (I use Default High Contrast theme, but I've tried several other themes, there seems no change. Any other extensions disabled.)
![image](https://user-images.githubusercontent.com/4270240/233850643-9c87c751-0d6b-4eed-859a-e2848c7d02b1.png)

User's settings.json:
```json
{
    "telemetry.enableTelemetry": false,
    "telemetry.enableCrashReporter": false,
    "extensions.ignoreRecommendations": true,
    "update.mode": "none",
    "files.watcherExclude": {
        "**/target/**": true
    },
    "lldb.showDisassembly": "never",
    "workbench.startupEditor": "none",
    "terminal.integrated.scrollback": 50000,
    "editor.cursorWidth": 10,
    "editor.cursorSurroundingLines": 3,
    "editor.unfoldOnClickAfterEndOfLine": true,
    "bracket-pair-colorizer-2.depreciation-notice": false,
    "workbench.activityBar.visible": false,
    "workbench.editor.wrapTabs": true,
    "editor.fontSize": 18,
    "terminal.integrated.fontSize": 18,
    "window.menuBarVisibility": "toggle",
    "editor.guides.bracketPairs": true,
    "security.workspace.trust.untrustedFiles": "open",
    "terminal.integrated.shellIntegration.enabled": true,
    "window.title": "${dirty}${activeEditorMedium}${separator}${rootName}${separator}${appName}",
    "workbench.experimental.editor.dropIntoEditor.enabled": false,
    "debug.console.closeOnEnd": true,
    "workbench.editor.tabCloseButton": "off",
    "git-buttons": {
        "commit": true
    },
    "editor.rulers": [
        100
    ],
    "debug.showBreakpointsInOverviewRuler": true,
    "editor.cursorStyle": "block-outline",
    "editor.renderLineHighlight": "gutter",
    "workbench.editor.highlightModifiedTabs": true,
    "rust-analyzer.typing.autoClosingAngleBrackets.enable": true,
    "editor.fontFamily": "'Source Code Pro', 'monospace', monospace",
    "telemetry.telemetryLevel": "off",
    "codetour.recordMode": "pattern",
    "codetour.customTourDirectory": "",
    "bookmarks.keepBookmarksOnLineDelete": true,
    "bookmarks.sideBar.expanded": true,
    "window.autoDetectHighContrast": false,
    "window.restoreFullscreen": true,
    "workbench.editor.enablePreview": false,
    "editor.wordWrap": "bounded",
    "BUFchronicler.ffmpeg-binary": "/usr/bin/ffmpeg",
    "chronicler.ffmpeg-binary": "/home/pkehl/GIT/present_markdown_reveal.js/all_args.sh",
    "chronicler.dest-folder": "${workspaceFolder}/.videos",
    "chronicler.recording-defaults": {
        "flags": {
            "c:v": "libx264rgb",
            "crf": 0,
            "color_range": 2
        },
        "fps": 4,
        "countdown": 3
    },
    "vscrof.directoryStyle": "Linux",
    "vscrof.storageFileTitle": "VsCode - Remember Open Files (vscrof)",
    "tab-groups.relativePaths": true,
    "tab-groups.saveGlobally": false,
    "editor.minimap.enabled": false,
    "sort-tabs.showStatusbarMessage": false,
    "errorLens.statusBarMessageEnabled": true,
    "errorLens.statusBarColorsEnabled": true,
    "errorLens.scrollbarHackEnabled": true,
    "rust-analyzer.cargo.unsetTest": [
        "alloc"
    ],
    "git.detectSubmodules": false,
    "statusBarCustomColor.primaryBkColor": "#000000",
    "statusBarCustomColor.secondaryBkColor": "#000000",
    "breadcrumbs.filePath": "last",
    "breadcrumbs.symbolPath": "on",
    "filePathBar.pathStyle": "relative",
    "GitLink.defaultRemote": "origin",
    "line-length-checker.lineLength": 100,
    "editor.wordWrapColumn": 100,
    "git.openRepositoryInParentFolders": "always",
    "remote.autoForwardPorts": false,
    "typescript.tsserver.watchOptions": {},
    "workbench.statusBar.visible": false,
    "colorTabs.titleBackground": true,
    "window.zoomLevel": 2,
    "workbench.colorTheme": "Default High Contrast"
}
```

Workspace settings.json:
```json
{
    "colorTabs.config": [
        {
            "regex": "allows/.*"
        },
        {
            "regex": "allows_internals/.*"
        },
        {
            "regex": "allows_tests/.*"
        }
    ]
}
```

VS Code on current stable kernel 6.2.12-1-MANJARO on Manjaro (Arch Linux stable derivative), GNOME 43.4 with X11
```
Version: 1.77.3
Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710
Date: 2023-04-12T09:16:52.732Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Linux x64 6.2.12-1-MANJARO
Sandboxed: No
```

**Comment by peter-lyons-kehl** [comment]
(clone of https://github.com/coop-rs/allows)
1. under `allows/`:
![image](https://user-images.githubusercontent.com/4270240/233866064-7b9a0150-516e-47a3-9724-529885bb53ee.png)
```json
"workbench.colorCustomizations": {
        "tab.activeBorder": "#39D3A1",
        "titleBar.activeBackground": "#39D3A1",
        "titleBar.activeForeground": "#ffffff",
        "statusBar.background": "#39D3A1",
        "statusBar.foreground": "#ffffff"
    }
```

2. under `allows_internals/`:
![image](https://user-images.githubusercontent.com/4270240/233866111-ca2f9e82-a44c-4d31-9d40-83e0979e0931.png)
```json
"workbench.colorCustomizations": {
        "tab.activeBorder": "#8BC8EA",
        "titleBar.activeBackground": "#8BC8EA",
        "titleBar.activeForeground": "#000000",
        "statusBar.background": "#8BC8EA",
        "statusBar.foreground": "#000000"
    }
```

3. under `allows_tests/`:
![image](https://user-images.githubusercontent.com/4270240/233866133-92405c4b-686a-40ce-9fdf-2d75e1569f5e.png)
```json
"workbench.colorCustomizations": {
        "tab.activeBorder": "#6686BF",
        "titleBar.activeBackground": "#6686BF",
        "titleBar.activeForeground": "#ffffff",
        "statusBar.background": "#6686BF",
        "statusBar.foreground": "#ffffff"
    }
```

**Comment by oreporan** [comment]
Hey, currently coloring the actual background of the tab is not supported, It was merged in #61  but never deployed. I'll take care of it

Also, the title of the issue is "Title background not applied", but I think you're talking about "tab backrground" ?

**Comment by oreporan** [comment]
deployed `tabBackground` https://github.com/oreporan/color-tabs-vscode/commit/00d60b2278a661c42ebc7cae680abd431f641112
